app/views/bgeigie_imports/_head.html.erb

```diff
@@ -13,6 +13,14 @@
           <%- end -%>
         </li>
       <%- end -%>
+      <%- if !@bgeigie_import.awaiting_approval? && user_signed_in? && current_user.moderator? && !@bgeigie_import.done? -%>
```

Comment:
Yeah, we wouldn't approve a broken drive, so need to be able to try and "fix" it before approval.